molbe/lo.py

```diff
@@ -298,6 +296,37 @@ def localize(self, lo_method, mol=None, valence_basis='sto-3g',
                 self.lmo_coeff = functools.reduce(numpy.dot,
                                                   (self.W.T, self.S, self.C))            
 
+    elif lo_method in ['pipek-mezey','pipek', 'PM']:
+    
+        es_, vs_ = eigh(self.S)
+        edx = es_ > 1.e-15                
+        self.W = numpy.dot(vs_[:,edx]/numpy.sqrt(es_[edx]), vs_[:,edx].T)
+        
+        es_, vs_ = eigh(self.S)
+        edx = es_ > 1.e-15                
+        W_ = numpy.dot(vs_[:,edx]/numpy.sqrt(es_[edx]), vs_[:,edx].T)
+        if self.frozen_core:                    
+            P_core = numpy.eye(W_.shape[0]) - numpy.dot(self.P_core, self.S)
+            C_ = numpy.dot(P_core, W_)
+            Cpop = functools.reduce(numpy.dot,
+                                    (C_.T, self.S, C_))
+            Cpop = numpy.diag(Cpop)
+            no_core_idx = numpy.where(Cpop > 0.55)[0]
```

Comment:
Maybe I'm missing something, but where does the 0.55 value come from?

Reply:
This is just the cutoff used elsewhere in the .lo module for defining occupieds. See below for more examples.